### PR TITLE
Add info tooltip to a ticket page

### LINKED ISSUE #428: add info tooltip to a ticket page
The screenshot below shows an info tooltip displayed when the mouse pointer is hovered over the "LIVE" text on the ticket card. The tooltip widget already exists in the godcr. It should also be added to the ticket page. The mockups should be referenced to see the text and how it is displayed for other tickets than the live ticket. 

Screenshot 2021-05-21 at 12 23 29

### PR DESCRIPTION
**Pull request by JustinBeBoy**
This PR resolve #428 

This PR all tooltip in LIVE and info of ticket in ticket page

### CODE REVIEW

File: ui/components.go [hunk 1]
```diff
@@ -831,8 +831,84 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
...
+			message = "Missed tickets will be revoked to return the original ticket price to you."
+			message2 = "If a ticket is not revoked automatically, use the revoke button."
+		} else if t.Info.Status == "EXPIRED" {
+			title = "This ticket has not been chosen to vote within 40960 blocks, and thus expired. "
+			message = "Expired tickets will be revoked to return the original ticket price to you."
+			message2 = "If a ticket is not revoked automatically, use the revoke button."
+		} else if t.Info.Status == "REVOKED" {
+			title = "This ticket has been revoked."
+			message = "The ticket price will become spendable after 256 blocks (~20 hrs)."
+		}
```
Comment (oshorefueled) [inline ui/components.go]: a switch statement would be much cleaner here

File: ui/components.go [hunk 2]
```diff
@@ -831,8 +831,84 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
 	}
 }
 
+func ticketStatusTooltip(gtx C, c pageCommon, rect image.Rectangle, t *wallet.Ticket, tooltip *decredmaterial.Tooltip) {
+	inset := layout.Inset{}
+	tooltip.Layout(gtx, rect, inset, func(gtx C) D {
+		st := ticketStatusIcon(&c, t.Info.Status)
+		title := ""
+		message := ""
+		message2 := ""
```
Comment (oshorefueled) [inline ui/components.go]:
```suggestion
		var title, message, message2 string
```

File: ui/components.go [hunk 3]
```diff
@@ -831,8 +831,84 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
...
+			title = "This ticket has been revoked."
+			message = "The ticket price will become spendable after 256 blocks (~20 hrs)."
+		}
+		titleLabel := c.theme.Label(values.MarginPadding14, title)
+		messageLabel := c.theme.Label(values.MarginPadding14, message)
+		messageLabel2 := c.theme.Label(values.MarginPadding14, message)
+		messageLabel.Color = c.theme.Color.Gray
+		messageLabel2.Color = c.theme.Color.Gray
+
+		status := c.theme.Label(values.MarginPadding14, t.Info.Status)
```
Comment (oshorefueled) [inline ui/components.go]:
```suggestion
		status := c.theme.Body2(t.Info.Status)
```

**Review by oshorefueled — CHANGES_REQUESTED** [review]

File: ui/components.go [hunk 4]
```diff
@@ -630,8 +630,86 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
...
+				if message2 != "" {
+					return layout.Inset{Top: values.MarginPadding8}.Layout(gtx, func(gtx C) D {
+						return messageLabel2.Layout(gtx)
+					})
+				}
+				return layout.Dimensions{}
+			}),
+		)
+	})
+}
```
Comment (Sirmorrison) [inline ui/components.go]: You have codes that are repeating `return layout.Inset{}`. see suggested below.


```suggestion
		st.icon.Scale = .5
		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
			layout.Rigid(func(gtx C) D {
				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
					layout.Rigid(st.icon.Layout),
					layout.Rigid(toolTipContent(gtx, layout.Inset{Left: values.MarginPadding4}, status.Layout)),
				)
			}),
			layout.Rigid(toolTipContent(gtx, layout.Inset{Top: values.MarginPadding8}, titleLabel.Layout)),
			layout.Rigid(toolTipContent(gtx, layout.Inset{Top: values.MarginPadding8}, messageLabel.Layout)),
			layout.Rigid(func(gtx C) D {
				if message2 != "" {
					toolTipContent(gtx, layout.Inset{Top: values.MarginPadding8}, messageLabel.Layout)
				}
				return layout.Dimensions{}
			}),
		)
	})
}

func toolTipContent(gtx layout.Context, inset layout.Inset, body layout.Widget) layout.Widget {
	return func(gtx C) D {
		return inset.Layout(gtx, body)
	}
}
```

File: ui/tickets_page.go [hunk 5]
```diff
@@ -265,7 +265,8 @@ func (pg *ticketPage) ticketsLiveSection(gtx layout.Context, c *pageCommon) layo
 				tickets := (*pg.tickets).LiveRecent
 				return pg.ticketsLive.Layout(gtx, len(tickets), func(gtx C, index int) D {
 					return layout.Inset{Right: values.MarginPadding8}.Layout(gtx, func(gtx C) D {
-						return ticketCard(gtx, c, &tickets[index])
+						tooltip := c.theme.Tooltip()
+						return ticketCard(gtx, c, &tickets[index], tooltip)
```
Comment (Sirmorrison) [inline ui/tickets_page.go]:
```suggestion
						return ticketCard(gtx, c, &tickets[index], c.theme.Tooltip())
```

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/components.go [hunk 6]
```diff
@@ -630,8 +630,78 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
...
+
+		status := c.theme.Body2(t.Info.Status)
+		status.Color = st.color
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						st.icon.Scale = .5
+						return st.icon.Layout(gtx)
+					}),
```
Comment (Sirmorrison) [inline ui/components.go]: You seem to have skipped this. kindly move `st.icon.Scale = .5` to line 675 or to line 641. it really does not need to be where it is.
```suggestion
					layout.Rigid(st.icon.Layout),
```

**Review by Sirmorrison — DISMISSED** [review]

**Review by oshorefueled — APPROVED** [review]

**Pull request merged**